CONTRIBUTING.rst

```diff
@@ -121,8 +121,8 @@ Ready to contribute? Here's how to set up `xclim` for local development.
 
 #. Create a development environment. We recommend using ``conda``::
 
-    $ conda create -n xclim python=3.8 --file=environment.yml
-    $ pip install -e .[dev]
+    $ conda create -n xclim python=3.9 --file=environment.yml
```

Comment:
```suggestion
    $ conda create -n xclim python=3.10 --file=environment.yml
```
As we are bumping the version, I would suggest recommending using the "middle" python version ? Makes for a longer supported environment. And brings more improvements.

Reply:
I have no strong opinions. Sounds good!